protected/jobHandlers/accessRequests/CreatedNotificationHandler.php

```diff
@@ -0,0 +1,47 @@
+<?php
+declare(strict_types=1);
+
+namespace prime\jobHandlers\accessRequests;
+
+use JCIT\jobqueue\interfaces\JobInterface;
+use prime\jobs\accessRequests\ResponseNotificationJob;
+use prime\models\ar\AccessRequest;
+use prime\models\ar\User;
+use yii\mail\MailerInterface;
+use function iter\rewindable\map;
+use function iter\toArray;
+
+class CreatedNotificationHandler extends AccessRequestHandler
```

Comment:
This should not extend here; this is a class hierarchy for the purpose of code reuse.
Use a repository that is injected (best imo), or a trait.